packages/snaps-sdk/src/types/device.ts

```diff
@@ -0,0 +1,97 @@
+import type { Struct } from '@metamask/superstruct';
+import { literal, refine, string, union } from '@metamask/superstruct';
+
+import type { Describe } from '../internals';
+
+/**
+ * The type of the device.
+ */
+export type DeviceType = 'hid' | 'bluetooth';
```

Comment:
Can we use an enum? https://github.com/MetaMask/snaps/blob/fb/devices/packages/snaps-controllers/src/devices/DeviceController.ts#L50-L52

Reply:
We could, but I don't see the benefit in this case.